internal/server/cache/cache.go

```diff
@@ -54,6 +55,8 @@ type Cache struct {
 	sqlProvenances map[string]*pb.Facet
 	// SQL database entity, variable existence pairs
 	sqlExistenceMap map[util.EntityVariable]struct{}
+	// SV -> inputPropertyExpressions for StatisticalCalculations.
```

Comment:
maybe use more verbose description for "->"? Also the value is a list of strings but the comment seems to be an object, clarify?

Reply:
Updated description